networking/eth2/src/test/java/tech/pegasys/artemis/networking/eth2/gossip/topics/Eth2TopicHandlerTest.java

```diff
@@ -28,13 +28,17 @@
 import org.junit.jupiter.api.Test;
 import tech.pegasys.artemis.datastructures.operations.Attestation;
 import tech.pegasys.artemis.datastructures.util.DataStructureUtil;
+import tech.pegasys.artemis.ssz.SSZTypes.Bytes4;
+import tech.pegasys.artemis.storage.client.RecentChainData;
 
 public class Eth2TopicHandlerTest {
```

Comment:
Do we have any tests around the topic format?  Probably worth adding a test here to make sure the topic looks like what we expect.

Reply:
Done.